code/game/objects/items/weapons/grenades/grenade.dm

```diff
@@ -108,3 +108,7 @@
 /obj/item/grenade/attack_hand()
 	walk(src, null, null)
```

Comment:
What's the reason for this walk to begin with?

Reply:
Yep okay I've looked at the code there. That's cursed. Can you add a comment here and to the Destroy line which explains why it is there?

Reply:
> Yep okay I've looked at the code there. That's cursed. Can you add a comment here and to the Destroy line which explains why it is there?

Done, please look over the comments and see if they're suitable